core/domain/platform_feature_services.py

```diff
@@ -270,13 +270,9 @@ def _evaluate_feature_flag_values_for_context(
 
     result_dict = {}
     for feature_name in feature_names_set:
-        feature = registry.Registry.get_platform_parameter(
+        param = registry.Registry.get_platform_parameter(
             feature_name)
-        assert feature.feature_stage is not None
-        if not feature_flag_valid_on_current_server(feature.feature_stage):
-            result_dict[feature_name] = False
-            continue
-        feature_name_value = feature.evaluate(context)
+        feature_name_value = param.evaluate(context)
```

Comment:
Why do we call this feature_name_value rather than feature_value (or feature_is_enabled)? What is the "value" of a "feature name"?

Reply:
Done